Bottom: 3d7a6fcb359026efffafa848528e8741f82a1d68
Top:    24d891d726e8f201f56bae8687c20b404515ca41
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-29 17:37:14 +0100

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon.c b/src/polygon.c
index fbc0c29..c3274c1 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -109,7 +109,8 @@ dicer output is used for HIDs which cannot render things with holes
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
 
-#define UNSUBTRACT_BLOAT MIL_TO_COORD (0.1)
+//#define UNSUBTRACT_BLOAT MIL_TO_COORD (0.1)
+#define UNSUBTRACT_BLOAT 10
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 1 /*100*/
 #define SUBTRACT_LINE_BATCH_SIZE 1 /*20*/
 
diff --git a/src/polygon1.c b/src/polygon1.c
index f3faba3..994aeff 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1663,9 +1663,11 @@ Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
       jump_cur_temp = cur;
       jump_dir_temp = dir;
 
+#if 1
       /* see where to go next */
       if (!jump (&jump_cur_temp, &jump_dir_temp, v_rule))
 	break;
+#endif
 
 #if 0
       /* see where to go next */
@@ -1693,8 +1695,15 @@ Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
 #ifdef DEBUG_GATHER
       DEBUGP ("gather vertex at %mm, %mm, Dir=%i\n", cur->point[0], cur->point[1], dir);
 #endif
+#if 1
       cur = jump_cur_temp;
       dir = jump_dir_temp;
+#endif
+#if 0
+      /* see where to go next */
+      if (!jump (&cur, &dir, v_rule))
+	break;
+#endif
 
       /* Now mark the edge as included.  */
       newn = (dir == FORW ? cur : cur->prev);
